evap/evaluation/templates/notebook.html

```diff
@@ -27,3 +27,15 @@ <h5 class="card-title m-0">
         </div>
     </div>
 </div>
+
+<div class="collapse show" tabindex="-1" id="notebookButton">
+    <button class="btn btn-outline-secondary button-open-notebook" type="button" data-bs-toggle="collapse" data-bs-target="#notebook">
+        <span class="fas fa-pen-to-square"></span>
+    </button>
+</div>
+
+<script type="module">
+    import  { initNotebook } from "{% static 'js/notebook.js' %}"
+    initNotebook("#notebook", "#notebook-form", "#evapContent", "evap_notebook_open", "#notebookButton");
```

Comment:
Can we pass `Element`s/`HTMLElement`s instead of selectors? I'd say that would be a more canonical interface.